bin/mr_bump

```diff
@@ -55,6 +55,12 @@ loop do
     MrBump.file_prepend('CHANGELOG.md', md_changes)
     system 'nano CHANGELOG.md'
     loop do
+      log = File.open('CHANGELOG.md', 'r').read
+      md_changes = log.split("\n\n").first.split("\n").drop(1)
```

Comment:
Im thinking perhaps we can write to a temporary file that contains just the change log items without the header. That way there is nothing to parse we can just read the file.